uspace/dist/src/c/demos/tetris/tetris.c

```diff
@@ -68,17 +68,17 @@ static volatile const char copyright[] =
 
 cell board[B_SIZE];
 
-int Rows;
-int Cols;
+errno_t Rows;
+errno_t Cols;
```

Comment:
Rows and Cols are rows and cols, not errnos. Also, this is a copy of the actual file, so I guess the same issue exists in the original.

Reply:
Actually, the issue is caused by it being an unbuilt copy. I forgot to correct the "demos" because they are not compiled. In fact, it seems wrong to have copies like this in the repository. What's up the reason for not copying it in the build script? This way, you have to manually copy over any changes to the original.

Reply:
A good point actually, I don't know why :-)